src/r-packages/devcontainer-feature.json

```
@@ -1,7 +1,7 @@
 {
 	"name": "R packages (via pak)",
 	"id": "r-packages",
-	"version": "1.0.3",
+	"version": "1.1.3",
```

Sorry, I wanted to say patch version instead of minor version...

```suggestion
	"version": "1.0.4",
```